tests/conftest.py

```diff
@@ -420,6 +420,8 @@ def factory(
         new_rbd_pool=False,
         pool_name=None,
         rbd_thick_provision=False,
+        encrypted=False,
```

Comment:
Please add `encrypted` and `encryption_kms_id` as part of Args description

Reply:
Done